bgcval2/download_from_mass.py

```diff
@@ -418,6 +422,7 @@ def download_from_mass(
     header_lines.append('# from login1.jasmin.ac.uk, ssh to the mass machine:\n#     ssh -X  mass-cli\n')
     header_lines.append(''.join(['# run script with:\n# source ', os.path.abspath(download_script_path),'\n']))
     header_lines.append('# moo passwd -r # if mass password is expired\n')
+    header_lines.append('source /etc/bashrc\n') # make sure script can access moo tools.
```

Comment:
is this a script that lives on `/etc` on mass-cli node?

Reply:
Yeah, its only about 90 lines long, but it's a bit too bash-y for me to understand. I'm happy to follow advice here instead of trying to DIY it! This method ensures that it stays up to date if the system updates /etc/bashrc.

Reply:
> This method ensures that it stays up to date if the system updates

Yes, good point! There's that, which I thought about too yesterday. Maybe I should have a closer look at the JASMIN setup and have a chat done with Alan/Ag about this